modules/store/src/selector.ts

```diff
@@ -552,6 +552,84 @@ export function createSelectorFactory<T = any, Props = any, V = any>(
   memoize: MemoizeFn,
   options: SelectorFactoryConfig<T, V>
 ): (...input: any[]) => MemoizedSelectorWithProps<T, Props, V>;
+/**
+ *
+ * @param memoize The the function used to memoize selectors
+ * @param options Config Object that may include a stateFn function defining how to return the selector's value, given the entire `Store`'s state, parent `Selector`s, `Props`, and a `MemoizedProjection`
```

Comment:
Because the others are also wrapped in a code block

```suggestion
 * @param options Config Object that may include a `stateFn` function defining how to return the selector's value, given the entire `Store`'s state, parent `Selector`s, `Props`, and a `MemoizedProjection`
```